src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -4262,7 +4262,7 @@ internal static IEnumerable<CompletionResult> CompleteFilename(CompletionContext
                             {
                                 var sessionStateInternal = executionContext.EngineSessionState;
                                 completionText = sessionStateInternal.NormalizeRelativePath(path, sessionStateInternal.CurrentLocation.ProviderPath);
-                                string parentDirectory = ".." + Path.DirectorySeparatorChar;
+                                string parentDirectory = ".." + StringLiterals.DefaultPathSeparator;
```

Comment:
@daxian-dbw Why we have and keep `DefaultPathSeparator` duplication?

Reply:
I don't know :) maybe just some legacy reasons.

Reply:
I opened #6373 to track this.